ci/check-manifest.sh

```diff
@@ -40,8 +40,19 @@ rm "${YAMLS[@]}"
 make manifest
 diff="$(git status --porcelain ${YAMLS[@]})"
 
-if [ ! -z "$diff" ]; then
+MULTICLUSTER_YAMLS=(
+    "multicluster/build/yamls/antrea-multicluster-leader-global.yml"
+    "multicluster/build/yamls/antrea-multicluster-leader-namespaced.yml"
+    "multicluster/build/yamls/antrea-multicluster-member.yml"
+)
+
+rm "${MULTICLUSTER_YAMLS[@]}"
+cd multicluster;make manifests;cd ..
```

Comment:
Nit: looks better to add a space after `;`

Reply:
Done.